scripts/migrate_to_rapidpro/upload_to_rapidpro.py

```diff
@@ -164,7 +164,7 @@ def string_field(field, value):
             VALUES %s
             ON CONFLICT DO NOTHING
             """,
-            ((contact_id, f"whatsapp:{wa}", wa, None, "whatsapp", ORG, 50, None, None),),
+            ((contact_id, f"whatsapp:{wa}", wa, None, "whatsapp", 1, 50, None, None),),
```

Comment:
I think this change is unrelated, and something we want to keep. Try merging in `develop` into this branch, and see if that fixes it.

Reply:
okay, pulled the develop version. But there is a flake8 error with that line being too long so I'll push those changes